service/src/main/java/uk/gov/hmcts/reform/fpl/service/DirectionHelperService.java

```diff
@@ -103,10 +209,35 @@ public void persistHiddenDirectionValues(List<Element<Direction>> directionWithV
      */
     public Map<String, List<Element<Direction>>> sortDirectionsByAssignee(List<Element<Direction>> directions) {
         return directions.stream()
-            .filter(x -> x.getValue().getCustom() == null)
             .collect(groupingBy(directionElement -> directionElement.getValue().getAssignee().getValue()));
     }
 
+    /**
+     * Removes directions where custom is set to Yes.
+     *
+     * @param directions any list of directions.
+     * @return a list of directions that are not custom.
+     */
+    public List<Element<Direction>> removeCustomDirections(List<Element<Direction>> directions) {
+        return directions.stream()
+            .filter(element -> !"Yes".equals(element.getValue().getCustom()))
+            .collect(Collectors.toList());
```

Comment:
could we have the _Yes_ and _No_ constants in a separate class?

Reply:
don't see any pros to doing this

Reply:
Yes I actually like this, Constants for EMPTY and SPACE are the most annoying for me ever. 

That Yes too much rooted in our CCD model. But to be honest I think that should be converted to boolean - probably we should have some of our Jackson converter or something like that. Boolean field marked with @YerOrNo annotation, what do you think?